app/src/main/res/values/colors.xml

```diff
@@ -59,5 +62,5 @@
     <color name="bg_bottom_sheet">#46ffffff</color>
 
     <color name="call_buttons_background">#BF999999</color>
-
+    <color name="favorite_icon_tint">#FFCC00</color>
```

Comment:
Finally ⭐

Reply:
Jupp :)

Reply:
Reason I did all this: Message loading and posting is broken on `master-broken` where the UI is nice, master works but UI needs polishing. So backport changes to have a working _and_ nice client on Android :)